Dockerfile

```diff
@@ -29,7 +29,8 @@ RUN set -x \
     && rm -r "$GNUPGHOME" /usr/local/bin/gosu.asc \
     && chmod +x /usr/local/bin/gosu \
     && gosu nobody true \
-    && apt-get -y autoremove
+    && apt-get -y autoremove \
```

Comment:
Possible to apt remove wget before this line? I should have removed wget after using it

Reply:
ok done